h2/src/main/org/h2/command/Parser.java

```diff
@@ -1572,6 +1572,22 @@ private Prepared parseShow() {
         } else if (readIf("DATESTYLE")) {
             // for PostgreSQL compatibility
             buff.append("'ISO' DATESTYLE");
+        } else if (readIf("SEARCH_PATH")) {
+            // for PostgreSQL compatibility
+            String[] searchPath = session.getSchemaSearchPath();
+            if (searchPath == null || searchPath.length == 0) {
+                StringUtils.quoteStringSQL(buff, null);
```

Comment:
If you want to return null here, use `buff.append("CAST(NULL AS VARCHAR)")`. Maybe empty string (`buff.append("''")`) should be used instead, I don't know how to test it in the real PostgreSQL. I don't see any way to clear the path in it.

Reply:
I just leave empty string for `session.getSchemaSearchPath() == null`, although this seems never happen.